src/test/java/com/juniorstart/juniorstart/service/ProjectServiceTest.java

```diff
@@ -0,0 +1,186 @@
+package com.juniorstart.juniorstart.service;
+
+import com.juniorstart.juniorstart.model.*;
+import com.juniorstart.juniorstart.model.audit.UserStatus;
+import com.juniorstart.juniorstart.payload.ProjectRequest;
+import com.juniorstart.juniorstart.payload.UserSummary;
+import com.juniorstart.juniorstart.repository.ProjectRepository;
+import com.juniorstart.juniorstart.repository.TechnologiesRepository;
+import com.juniorstart.juniorstart.repository.UserDao;
+import com.juniorstart.juniorstart.security.UserPrincipal;
+import com.juniorstart.juniorstart.util.MockUtil;
+import com.nimbusds.oauth2.sdk.http.HTTPResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ExtendWith(MockitoExtension.class)
+public class ProjectServiceTest {
+
+    ProjectRepository projectRepository = Mockito.mock(ProjectRepository.class);
+    UserDao userDao = Mockito.mock(UserDao.class);
+    TechnologiesRepository technologyRepository = Mockito.mock(TechnologiesRepository.class);
+
```

Comment:
@InjectMocks @Mock

Reply:
Na to samo wychodzi.

Reply:
Tak ale po to są adnotacje zrobione żeby kod był czytelniejszy i żeby nie iść trudniejsza drogą.